please_wait.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Unidling, please wait</title>
+  </head>
+  <body>
+    <h1>Unidling, please wait a few seconds</h1>
+    <script>
+(function () {
+
+  var delay = 3000;
+  var url = 'UNIDLER_REDIRECT_URL';
+
+  function redirect_when_ready() {
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', url, true);
+
+    xhr.onreadystatechange = function () {
+      if (xhr.readyState === 4) {
```

Comment:
I confess I had to look at the MDN documentation.

It seems like [there is a `XMLHttpRequest.DONE` constant](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/onreadystatechange#Example), feel free to use it instead of the `4` which is a bit magical (and also, you have the other win of not needing the comment below to explain the `4` stands for :) )

Reply:
That is exactly what I want, thanks.